lldb/source/Plugins/TypeSystem/Clang/TypeSystemClang.cpp

```diff
@@ -2771,8 +2771,14 @@ static bool GetCompleteQualType(clang::ASTContext *ast,
         ast, llvm::cast<clang::AttributedType>(qual_type)->getModifiedType(),
         allow_completion);
 
-  case clang::Type::MemberPointer:
+  case clang::Type::MemberPointer: {
+    auto *MPT = qual_type.getTypePtr()->castAs<clang::MemberPointerType>();
+    if (MPT->getClass()->isRecordType())
+      GetCompleteRecordType(ast, clang::QualType(MPT->getClass(), 0),
+                            allow_completion);
```

Comment:
Technically we need to run the completion only on MS-ABI. Though I *think* this shouldn't be too expensive since member pointers aren't a very common code path, so probably not worth guarding this on the target ABI. But should we add a comment here saying we need this type to be completed because of MS-ABI?

Reply:
That right! In fact, it was quite simple, so I did both.

Reply:
thanks!